client/src/main/java/com/vesoft/nebula/client/graph/NGQLException.java

```diff
@@ -10,15 +10,20 @@
  *
  */
 public class NGQLException extends Exception {
+    private static final long serialVersionUID = -90563191494716743L;
```

Comment:
`serialVersionUID` is used to serialization and deserialization. Don't add for each class.